crimsobot/utils/games.py

```diff
@@ -82,57 +86,123 @@ async def win(discord_user: DiscordUser, amount: float) -> None:
     account.add_to_balance(amount)
     await account.save()
 
+# simple_eval() by MestreLion via StackOverflow with changes in exception handling
+# https://stackoverflow.com/a/65945969
+
+# Kept outside simple_eval() just for performance
+_re_simple_eval = re.compile(rb'd([\x00-\xFF]+)S\x00')
```

Comment:
notes from looking into this:
`d` = `LOAD_CONST` (decimal: 100, hex: 0x64, ascii: d)
two bytes specifying the index of the const, uses system endianness
`S` = `RETURN_VALUE` (decimal: 83, hex: 0x53, ascii: S)
null-terminated